aws/main.tf

```diff
@@ -17,6 +17,10 @@ module "redpanda-cluster" {
   associate_public_ip_addr = var.associate_public_ip_addr
   availability_zone        = var.availability_zone
   client_count             = 1
+  broker_instance_type     = var.instance_type
+  client_instance_type     = var.instance_type
+  prometheus_instance_type = var.instance_type
```

Comment:
Was this removed at some point?

I remember using `instance_type` previously, and even the docs mention it: https://docs.redpanda.com/22.1/deployment/production-deployment-automation/#step-3-create-the-cluster

Reply:
I removed it previously because this is just an example element to show one possible implementation of the module -- and for CI -- and the module actually has three separate instance types defined. But it saves me a little bit of typing and keeps us 1:1 with the docs soo... 😅